core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -1108,6 +1114,11 @@
    */
   public static final String CARBON_DATA_FILE_DEFAULT_VERSION = "V3";
 
+  /**
+   * current CarbonData version
+   */
+  public static final String CARBONDATA_DEFAULT_VERSION = "1.3.0-SNAPSHOT";
```

Comment:
here, it is better to automatically get version number from pom file.

Reply:
I also think so, but I don't know how to implement it. I will try.

Reply:
I think you can do it as the same as spark, it generates a file called 'spark-version-info.properties' when build, and then read version information from this file at runtime.

Reply:
Ok, I will try. Thank you.

Reply:
@zzcclp It has a little complexity in spark, is there better solution?

Reply:
I think it's a good way to do like spark, it's not complexity , I can help you to implement this.

Reply:
Ok, you can do it.

Reply:
@zzcclp  I assign the jira to you, and close this PR.